wp-content/themes/core/components/dialog/Dialog_Controller.php

```diff
@@ -0,0 +1,152 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\dialog;
+
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Project\Templates\Components\Deferred_Component;
+use Tribe\Project\Templates\Components\container\Container_Controller;
+use Tribe\Project\Templates\Components\text\Text_Controller;
+
+class Dialog_Controller extends Abstract_Controller {
```

Comment:
Minor but there are some white space / alignment issues here that should be cleaned up before merging.